corehq/apps/sms/static/sms/js/sms_language_main.js

```diff
@@ -0,0 +1,7 @@
+hqDefine("sms/js/sms_language_main",[
+    "langcodes/js/langcodes",
```

Comment:
[This file](https://github.com/dimagi/langcodes/blob/master/static/langcodes/js/langcodes.js) still depends on a global `jQuery` and on having `select2` available. Can you update it to be AMD-friendly?

Reply:
Curious, does this work? I'd expect that the jQuery is okay (the global is still technically available on requirejs pages) but that the select2 call would fail.

Reply:
Ugh, caching can be such a pain sometimes. Testing on staging is definitely a good practice for requirejs work.

Reply:
https://github.com/dimagi/langcodes/pull/23